app/javascript/app/acuant/document_capture_dom.js

```diff
@@ -28,6 +29,16 @@ const hideAcuantSdkContainers = () => {
   acuantSdkContinueForm().classList.add('hidden');
 };
 
+export const hideAcuantSdkCaptureView = () => {
```

Comment:
A bit of a nit, but wdyt of changing this to match the naming patter for `acuantImageCaptureStarted` e.g. `acuantImageCaptureEnded()`?

Reply:
Ok. No problem.